.github/workflows/main.yml

```diff
@@ -36,7 +36,7 @@ jobs:
         with:
           node-version: "16"
           cache: "npm"
-      - run: yarn install --frozen-lockfile
+      - run: yarn install
```

Comment:
```suggestion
      - run: yarn install --frozen-lockfile
```

Or for Yarn v3:

```suggestion
      - run: yarn install --immutable
```

Reply:
The yarn lock files need to be updated, but that should be part of your commit. By the time CI is running, the yarn lock file shouldn't need to be updated (this is basically the use-case for this flag).